TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -130,21 +129,17 @@ class RouteOptionsViewController: UIViewController {
 
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
-        setupReachability()
         setUpRouteRefreshing()
     }
 
     override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
-        // Takedown reachability
-        reachability?.stopNotifier()
-        NotificationCenter.default.removeObserver(self, name: .reachabilityChanged, object: reachability)
         // Remove banner
         banner?.dismiss()
         banner = nil
         routeTimer?.invalidate()
         updateTimer?.invalidate()
-        // Stop observing when app becomes active 
+        // Stop observing when app becomes active
```

Comment:
I'm a little confused by this comment. Doesn't the next line stop observing when the view controller is dismissed?

Reply:
+1

Reply:
Yea, I agree. The comment doesn't make sense. I think the person that put in the comment meant to say stop observing for the notification called `appBecameActiveNotification`. But will fix that!